a3p-integration/proposals/f:fast-usdc/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "agoricProposal": {
+    "source": "subdir",
+    "sdk-generate": [
+      "fast-usdc/init-fast-usdc.js submission --oracleSet A3P_INTEGRATION"
+    ],
+    "type": "/agoric.swingset.CoreEvalProposal"
+  },
+  "type": "module",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/client-utils": "0.1.1-dev-02c06c4.0",
```

Comment:
this should just be `dev`. the specific sha will be in `yarn.lock`.

consider integrating with the latest in `agoric-sdk` using `yarn link` https://github.com/Agoric/agoric-sdk/blob/dcd05b441beb52656411ec0d33d7ed6484ff6b14/a3p-integration/README.md#build-details